cmd/power.go

```
@@ -247,7 +250,7 @@ func init() {
 	PowerCmd.Flags().String("cacert", "", "Set the path to CA cert file (defaults to system CAs when blank)")
 	PowerCmd.Flags().VarP(&powerFormat, "format", "F", "Set the output format (json|yaml)")
 
-	PowerCmd.RegisterFlagCompletionFunc("format", completionFormatData)
+	_ = PowerCmd.RegisterFlagCompletionFunc("format", completionFormatData)
```

See comment above regarding ignoring errors.